Install the deps with error '404 Not Found: serve@^6.4.8'

tinys-MacBook-Pro:rollup-starter-app tinychou$ npm i
WARN tar ENOENT: no such file or directory, open '/Users/tinychou/Public/git_repo/rollup-starter-app/node_modules/.staging/is-extglob-3003c40a/package.json'
WARN tar ENOENT: no such file or directory, open '/Users/tinychou/Public/git_repo/rollup-starter-app/node_modules/.staging/is-extglob-3003c40a/README.md'
WARN tar ENOENT: no such file or directory, open '/Users/tinychou/Public/git_repo/rollup-starter-app/node_modules/.staging/is-extglob-3003c40a/LICENSE'
WARN tar ENOENT: no such file or directory, open '/Users/tinychou/Public/git_repo/rollup-starter-app/node_modules/.staging/is-extglob-3003c40a/index.js'
npm WARN rollup-starter-app@ No repository field.
npm WARN rollup-starter-app@ No license field.

npm ERR! code E404
npm ERR! 404 Not Found: serve@^6.4.8

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/tinychou/.npm/_logs/2018-06-02T01_26_57_948Z-debug.log

--- a/package.json
+++ b/package.json
@@ -1,15 +1,15 @@
 {
   "name": "rollup-starter-app",
   "devDependencies": {
-    "@rollup/plugin-commonjs": "^11.0.1",
-    "@rollup/plugin-node-resolve": "^7.0.0",
+    "@rollup/plugin-commonjs": "^17.0.0",
+    "@rollup/plugin-node-resolve": "^11.1.0",
     "npm-run-all": "^4.1.5",
-    "rollup": "^1.16.2",
-    "rollup-plugin-terser": "^5.0.0",
-    "serve": "^11.0.2"
+    "rollup": "^2.36.2",
+    "rollup-plugin-terser": "^7.0.2",
+    "serve": "^11.3.2"
   },
   "dependencies": {
-    "date-fns": "^1.30.1"
+    "date-fns": "^2.16.1"
   },
   "scripts": {
     "build": "rollup -c",

